MikeSusutZZ/Project-Quazar
# Create high scores component; update game.ex logic to retrieve high-score data

## Summary
- Converted the `all-time-high-scores` LiveView Page into a Live Component.
- Added component to `components` directory
- Updated `game.html.heex` to render component in game lobby
- Updated `game.ex` to subscribe to `high_scores:updates` PubSub, which delivers data stored in an ETS (big thanks to @shawnbirring for setting this up)

This page will need some styling and organization, but I wanted to get the functionality up and running.

@mikkosio, can you double-check that I updated `game.ex` correctly, and that I didn't interfere with or change the functionality of your code?

## Testing
- Check out my branch: `create-high-scores-component`
- Open a terminal at the `project_quazar` folder
- run `iex -S mix phx.server`
- Go to `localhost:4000`; you should see an "All-Time High Scores" header
- In the terminal, enter the following command: 

```
ProjectQuazar.HighScores.ETSWrapper.insert_entry("Player1", 100)
```
- Confirm the entry appears under "All-Time High Scores" in the game lobby without needing to refresh the page.
